common/v2/features/SendAssets/components/fields/EthAddressField.tsx

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import { Input } from '@mycrypto/ui';
-import { Field, FieldProps } from 'formik';
+import { FieldProps, Field } from 'formik';
 
 import { getENSTLDForChain } from 'libs/ens/networkConfigs';
```

Comment:
It would be nice to gradually remove any dependencies from legacy code.
Can we import these from `v2` please.